repository/src/main/java/org/alfresco/repo/download/ZipDownloadExporter.java

```diff
@@ -137,13 +144,30 @@ public void start(final ExporterContext context)
     public void startNode(NodeRef nodeRef)
     {
         this.currentName = (String)nodeService.getProperty(nodeRef, ContentModel.PROP_NAME);
+        Date createdUtcTimestamp = (Date)nodeService.getProperty(nodeRef, ContentModel.PROP_CREATED);
+        Date modifiedUtcTimestamp = (Date)nodeService.getProperty(nodeRef, ContentModel.PROP_MODIFIED);
+        TimeZone time_zone_default = TimeZone.getDefault();
+        SimpleDateFormat formatter= new SimpleDateFormat("dd MMM yyyy HH:mm:ss z");
+        formatter.setTimeZone(time_zone_default);
+        try
+        {
+            this.zipTimestampCreated = formatter.parse(formatter.format(createdUtcTimestamp));
+            this.zipTimestampModified = formatter.parse(formatter.format(modifiedUtcTimestamp));

```

Comment:
What is this about?? Why are we running the date through the formatter twice?

Reply:
This is all for converting the UTC time to local time and we are using formatter twice because the Format method returns string and we are converting to date via parse method. 
As date is required for the setting time in the apache archive library.